justfile

```diff
@@ -31,7 +31,7 @@ virtualenv:
     PYTHON_VERSION=${PYTHON_VERSION:-$DEFAULT_PYTHON}
 
     # create venv and install latest pip that's compatible with pip-tools
```

Comment:
Is the comment `latest pip that's compatible with pip-tools` not relevant any more?

Reply:
Good spot; no, it's not really relevant, so just added a commit to remove it.